hdfs3/core.py

```diff
@@ -7,7 +7,8 @@
 import warnings
 import fnmatch
 PY3 = sys.version_info.major > 2
-here = os.path.dirname(os.path.abspath(__file__))
+# here = os.path.dirname(os.path.abspath(__file__))
+here = '/usr/local/lib'
```

Comment:
Probably the confusion was just around the name `here` rather than something like `where_so_files_live` (only more terse)